lib/worker/branch_deleter.ex

```diff
@@ -66,8 +68,19 @@ defmodule BorsNG.Worker.BranchDeleter do
       _ -> false
     end
 
-    if pr_in_same_repo && delete_merged_branches && pr.merged do
-      GitHub.delete_branch!(conn, pr.head_ref)
+    pr_closed = pr.state == :closed
+
+    pr_squash_merged = String.starts_with?(pr.title, "[Merged by Bors] - ")
+
+    if pr_in_same_repo && delete_merged_branches do
+      cond do
```

Comment:
Really not sure if it's the best way to do it, seemed more clear to me though.

Reply:
Seems fine to me.